### ISSUE TITLE
Feature request: Device page

### ISSUE THREAD

**Issue by Dinth** [cgarwood/homeassistant-zwave_mqtt#117]
Just another feature request - would it be possible to include node number and maybe an MQTT topic on Device pages in Zwave-Over-MQTT?

**Comment by marcelveldt** [comment]
Why?

The node ID can be found on entity level as attribute.
We're doing our best to keep this kind of advanced information as far away from users as possible to not overwhelm them.

But, maybe we can fill some of the device details with that info.

**Comment by cgarwood** [comment]
Device registry doesn't support custom fields at the moment :(

ZHA/Zigbee gets around that by storing their own cache file and using custom cards in their configuration UI.

Ideally I'd like to kill off node_id as an attribute on all the entities, but I want to wait until our new config panel is built before we do that.

**Comment by marcelveldt** [comment]
Yeah, let's work this out once we get the config panel in place.

**Comment by Dinth** [comment]
Fair enough

The justification is that node id is the only way of distinguishing which node is what in a bigger environment. Unfortunately i havent made a note of node_ids so this wouldnt help me in my configuration, but having a couple of dozens of switches, all hidden in walls controlling valves etc, makes really hard to identify whats what without a node id clearly visible in a place where nodes are renamed.

**Comment by marcelveldt** [comment]
But that is exactly why you can give personal names to devices ?

I have like 50+ zwave device but I have no clue which node_id is where. I just give them recognisable names.

**Comment by dkjsj** [comment]
How about with scenes? Right now the only way to find node's scene-event is by their node-id? Eg. listning to zwave_mqtt.scene_activated.

Any good ideas to solve that?

Example of scene event:

```
{
    "event_type": "zwave_mqtt.scene_activated",
    "data": {
        "node_id": 3,
        "scene_id": 4,
        "scene_label": "Scene 4",
        "scene_value_id": 0,
        "scene_value_label": "Inactive"
    },
    "origin": "LOCAL",
    "time_fired": "2020-04-30T19:55:03.328307+00:00",
    "context": {
        "id": "032164795ea54d92b19936a9974e0143",
        "parent_id": null,
        "user_id": null
    }
}
```

**Comment by marcelveldt** [comment]
@dkjsj this will be solved with device automations.

**Comment by dkjsj** [comment]
> @dkjsj this will be solved with device automations.

Looking forward to that :) This is one of the missing functionalty before I'll try this out in a larger scale (are right now using zwave2mqtt)

Right now I'm trying out some wall switches ([http://manuals-backend.z-wave.info/make.php?lang=en&sku=ZHC5010&cert=ZC10-16055065](url)) - and yes - I can capture the scene-event (1, 2 times pressed button) - but yeah device automations would be easier.

**Comment by Dinth** [comment]
> But that is exactly why you can give personal names to devices ?

Thats true when adding devices through Zwave-Over-MQTT and naming them as they are being added i believe, but if someone migrates existing Zwave mesh to Zwave-over-MQTT, or even mass adds the devices there will be just generic names (device vendor and model) set up, for few dozens of devices, additionally some nodes create two different devices with same names. Node ID is the only way of distinguishing those nodes from each other

**Comment by Quotacious** [comment]
Ideally, that would be handled by #19, so existing names would be migrated over. I shudder to think of having to rename everything again.

**Comment by tboyce021** [comment]
There's also still the case of moving the controller from a different platform where the devices are already paired but don't have names. In that case, they'll all just show up at once. That's essentially what it was like for me moving from zwave2mqtt since the existing names in HA were tied to the mqtt devices, not the zwave ones.

**Comment by Dinth** [comment]
Another suggestion, it would be nice to also see a device's firmware version if this information is exposed over MQTT.
